providers/terraform-provider-csbsqlserver/connector/url_encoder_test.go

```diff
@@ -0,0 +1,51 @@
+package connector_test
+
+import (
+	"net/url"
+	"reflect"
+	"testing"
+
+	"github.com/cloudfoundry/csb-brokerpak-aws/terraform-provider-csbsqlserver/connector"
+)
+
+func TestNewEncoder(t *testing.T) {
```

Comment:
any reason we are using the default test framwork instead of a Ginkgo DescribeTable here? I would favour consistency throughout the code unless there is a very good reason.